sixtyfps_compiler/generator/cpp.rs

```diff
@@ -953,7 +953,7 @@ fn generate_component(
             Access::Public,
             Declaration::Function(Function {
                 name: "window".into(),
-                signature: "() const -> const sixtyfps::Window&".into(),
+                signature: "() -> sixtyfps::Window&".into(),
```

Comment:
I think it's ok to return a non-const reference from a `const` function.
In this case, i'd say that the window function should be const because it does not change the component.

The same could be said about hide and show, they don't change the component, they could be const.

Reply:
Ah yes, you're right, I'll change the accessor to be `const` again. I'd like to keep `show()`/`hide()` and `run()` as non-const unless you object. `show()` and `hide()` change the visibility state (quite... visibly :-) and `run()` is all about changing the state.

Reply:
You're missing the `const_cast` 
(or the `mutable`)

Reply:
🤦‍♂️ You're totally right (as usual). My C++ is becoming rusty :-). Added as commit to this series now.